src/cljs/webchange/ui_framework/layout/navigation/styles.scss

```diff
@@ -1,42 +1,106 @@
-.navigation-menu {
-    .menu {
-        border-radius: 8px;
-        margin: 0;
-        padding: 0 0 0 32px;
-        width: 100%;
-        min-width: 200px;
-
-        .menu-item {
+.left-side-bar {
+    max-width: 296px;
+	width:100%;
+	height: 100%;
+	background: #353535;
+	padding:40px 24px 88px;
+	box-sizing: border-box;
+	position: fixed;
```

Comment:
Why is it `fixed`? I think it should be `relative`. And then `.content` block should have horizontal padding about 120px instead of 400px. Fix indents and spaces in the file, please (use spaces instead of tabs).